README.md

```diff
@@ -22,7 +22,7 @@ If you'd like to contribute, please review the
 To run tests locally:
 ```shell
 # Bootstrap a platform on a vagrant environment using
-make vagrantup
+doit vagrantup
```

Comment:
That won't work. While we can somewhat reasonably assume one has `make` available on a system, or can easily get it installed, chances are slim `doit` is pre-installed.

Either setting up the whole venv etc. could be added here (which is meh, and we'll run into issues where people created the venv, we update requirements, people don't re-run `pip install ...`, and things break), or we have `make` targets to set up the venv, or we introduce the `tox` based targets (wrapped by `make` `PHONY` targets?), or ...

Reply:
Yeah, shouldn't have changed that as we will still use a dummy Make to have short commands.

Reply:
Done